interface/portraits/portraits_tauren.gfx

```diff
@@ -12,8 +12,21 @@ spriteTypes = {
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
```

Comment:
I would say they're really bad quality.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105849780-493b0700-5ffa-11eb-9f06-8ecd45630674.png)
![image](https://user-images.githubusercontent.com/46384992/105850030-9d45eb80-5ffa-11eb-9ad6-bad34158c343.png)

</details>